### DISCUSSION TITLE
Enforcing data which is on entire domain as contraint to loss function

### DISCUSSION THREAD

**Discussion by dapyrate** [lululxvi/deepxde#724] [Q&A]
Basically I have `x`, `y`, and `z` values. You can suppose `x`, `y` and `z` are real world data. `x` and `y` are points on the domain and `z` is the value of the desired function.
I have written something like this: 
`observe_z = dde.icbc.PointSetBC(data_X, data_Z, component=0)`
and added the above line to 
`dde.data.pde(..., [bc, ic, observe_z],)`
Does this enforce `z` values associated with `x` and `y` on the entire domain during training?

**Comment by lululxvi** [comment]
Yes.
